description/newspaper.shacl.ttl

```diff
@@ -0,0 +1,170 @@
+        sh:description "Extra indication of the type of NewspaperType."@en ;
+        sh:description "Indication supplémentaire du type de NewspaperType."@fr ;
+        sh:description "Indicatie van het type NewspaperType."@nl ;
+        sh:message "the object of dct:format is not of type skos:Concept, is missing or occurs more than once"@en ;
+        sh:message "l'objet de dct:format n'est pas de type skos:Concept, est manquant ou apparaît plus d'une seule fois"@fr ;
+        sh:message "het object van dct:format is niet van het type skos:Concept, ontbreekt of komt meer dan eens voor"@nl ;
+        sh:severity sh:Violation 
+    ] ,
+    [
+        a sh:PropertyShape ;
+        sh:path haNePa:issuanceFrequency ;
+        sh:class skos:Concept ;
+        sh:minCount 0 ;
+        sh:maxCount 1 ;
+        sh:name "frequency of issuance"@en ;
+        sh:name "fréquence d'émission"@fr ;
+        sh:name "uitgavefrequentie"@nl ;
+        sh:description "Indicates how frequent a newspaper is issued, e.g. twice a day, daily, weekly etc."@en ;
+        sh:description "Indique la fréquence de parution d'un journal, par ex. deux fois par jour, quotidiennement, hebdomadairement, etc."@fr ;
+        sh:description "Geeft aan hoe vaak een krant verschijnt, b.v. tweemaal per dag, dagelijks, wekelijks etc."@nl ;
+        sh:message "the object of haNePa:issuanceType is not of type skos:Concept, or occurs more than once"@en ;
+        sh:message "l'objet de haNePa:issuanceType n'est pas de type skos:Concept, ou apparaît plus d'une seule fois"@fr ;
+        sh:message "het object van haNePa:issuanceType is niet van het type skos:Concept, of komt meer dan eens voor"@nl ;
+        sh:severity sh:Violation
+    ] ,
+    [
+        a sh:PropertyShape ;
+        sh:path haNePa:issueType ;
+        sh:class skos:Concept ;
+        sh:minCount 0 ;
+        sh:maxCount 1 ;
+        sh:name "publication type"@en ;
+        sh:name "type de publication"@fr ;
+        sh:name "uitgavetype"@nl ;
+        sh:description "Indicates the specific publication type of a newspaper edition (e.g. morning, evening, weekend newspaper edition, etc.)."@en ;
+        sh:description "Indique le type de publication spécifique d'une édition de journal (par exemple, édition du journal du matin, du soir, du week-end, etc.)."@fr ;
+        sh:description "Geeft het specifieke uitgavetype aan van een kranteneditie (bv. ochtend-, avond-, weekendkranteneditie etc.)."@nl ;
+        sh:message "the object of haNePa:issueType is not of type skos:Concept, or occurs more than once"@en ;
+        sh:message "l'objet de haNePa:issueType n'est pas de type skos:Concept, ou apparaît plus d'une seule fois"@fr ;
+        sh:message "het object van haNePa:issueType is niet van het type skos:Concept, of komt meer dan eens voor"@nl ;
+        sh:severity sh:Violation
+    ] .
+
+haNePa:NewspaperPageTypeNodeShape a sh:NodeShape ;
+    sh:targetClass haNePa:NewspaperPageType ;
+    sh:property [
```

Comment:
Add minCount 1 for the `schema:isPartOf` relationship to a NewspaperEdition? I know it is in the general datamodels, but it might seem weird that a newspaperpage is not connected to its edition.